ashraesizer.py

```diff
@@ -125,16 +125,53 @@ def primaryCurveAshrae(self, peakFlowTable):
         diffPeakFlowTable = np.insert(diffPeakFlowTable, len(diffPeakFlowTable)-1, diffPeakFlowTable[len(diffPeakFlowTable)-1], axis=0)
         
     
-        peakVolume = peakFlowTable[:,1] * self.People;
+        peakVolume = peakFlowTable[:,1] * self.nPeople
        
-        primaryVol = peakVolume / self.percentUseable * (self.supplyT - self.incomingT) / \
-                (self.storageT - self.incomingT);
+        primaryVol = peakVolume / self.percentUseable * (self.supplyT_F - self.incomingT_F) / \
+                (self.storageT_F - self.incomingT_F)
         
-        accurateRecoveryTons = self.People * (diffPeakFlowTable[:len(diffPeakFlowTable),1]) / \
-                (diffPeakFlowTable[:len(diffPeakFlowTable),0]) * 60 * self.rhoCp * (self.storageT - self.incomingT)/12000;
+        accurateRecoveryTons = self.nPeople * (diffPeakFlowTable[:len(diffPeakFlowTable),1]) / \
+                (diffPeakFlowTable[:len(diffPeakFlowTable),0]) * 60 * rhoCp * (self.storageT_F - self.incomingT_F)/12000
                 
         # simpleRecoveryTons = self.People * (peakFlowTable[:len(peakFlowTable),1]) / \
-        #         (peakFlowTable[:len(peakFlowTable),0]) * 60 * self.rhoCp * (self.storageT - self.incomingT)/12000;
+        #         (peakFlowTable[:len(peakFlowTable),0]) * 60 * rhoCp * (self.storageT_F - self.incomingT_F)/12000;
                 
-        return [primaryVol, accurateRecoveryTons];
-     
\ No newline at end of file
+        return [primaryVol, accurateRecoveryTons]
+     
+    def tonsRecoveryForMaxDaily(self, peakFlowTable, accurateRecoveryTons):
+        """"Calculates the simple tons required for recovery"""
+        
+        # Define the range of recovery hours
+        recoveryHours = np.array([1,2,4,8,12,16,24])
+        
+        simpleTons = np.amax(peakFlowTable[6,1])*self.nPeople*rhoCp*(self.supplyT_F-self.incomingT_F)/12000/recoveryHours
+        
+        # Find the recoveryHours below and above the defined compressor run time
+        lowerRecoveryHours = recoveryHours[recoveryHours < self.compRuntime_hr].max()
+        upperRecoveryHours = recoveryHours[recoveryHours > self.compRuntime_hr].min()
+        
+        # Find the indices surrounding the defined compressor run time
+        hoursIndices = np.where((recoveryHours == upperRecoveryHours) | (recoveryHours == lowerRecoveryHours))
+        
+        recoveryTimes = recoveryHours[hoursIndices]
+        recoveryTons = simpleTons[hoursIndices] 
```

Comment:
I think all of this is unnecessary. It looks like you're trying to pin point the indices greater than and less than recovery tons and hours, but np.interp will do this for you.

Reply:
You're right. I'll remove the unnecessary pieces.